### DISCUSSION TITLE
Lags on low end hardware (Results from my test) (Setting Menu feature suggestion)

### DISCUSSION THREAD

**Discussion by Codyboi123** [FunkinCrew/Funkin#1631] [Ideas]
(Do note I tested all of this on itch io version that's up right now) I check out the game on multiple hardware, higher end PC to low-end laptops. With my experimenting I found the game plays really choppy on the low-end laptop. (The low-end laptop has 4 GB of Ram and a integrated GPU in the CPU which is a AMD Quad-Core A6-7310 APU (2 GHz, up to 2.4 GHz, 2 MB cache, with AMD Radeon R4 Graphics) I played on the web on the itch.io page, and found it to play slow and choppy with 30 frames in the main menu with 1 to 10 frames while actually playing. I did change some hidden google chrome setting by enabling "override software rendering list", which made it run better with higher frames with 60 frames in the menu, but the actual gameplay, it's still choppy with it jumping between 10 to 50 frames, with sound being a bit off sync. I found this weird because the original game that release during the Ludum Dare prototype version on itch io runs at between 40 to 60 frames. It's only the new updated version on itch io that lags like this.    
**The main feature I am requesting** is if there is a way to create a setting that would lower the prosses such as turning off effects like dynamic camera movement and lowering the resolution of the graphics. However, the game on my main high-end PC ran smooth with the the infrequent dip in frames between loading.

**Comment by TentaRJ** [comment]
> With my experimenting I found the game plays really choppy on the low-end laptop.

## 🤯 

>  I played on the web on the itch.io page, and found it to play slow and choppy with 30 frames in the main menu with 1 to 10 frames while actually playing.

Maybe do your tests by running on the desktop build of the game and not the browser because you would be able to access more resources.

## Oh also, make sure you use the template for issues next time...
